packages/office-ui-fabric-react/src/components/Nav/Nav.tsx

```diff
@@ -144,8 +144,9 @@ export class Nav extends BaseComponent<INavProps, INavState> implements INav {
           [styles.compositeLinkIsExpanded]: link.isExpanded,
           [styles.compositeLinkIsSelected]: isLinkSelected
         }) }>
-        { (nestingLevel === 0 && link.links && link.links.length > 0 ?
+        { (link.links && link.links.length > 0 ?
           <button
+            style={ { marginLeft: `${14 * nestingLevel}px` } }
```

Comment:
Not rtl safe...

if you need to dynamically provide left margin, you need to use the getRTL helper to switch this around for rtl. (search the code for examples.)